doc.rst.j2

```diff
@@ -0,0 +1,29 @@
+###############
+Feature Toggles
+###############
+
+These are all of our well-documented feature toggles!
+
+{% for group in groups %}
+{{ group.feature_toggle_name }}
```

Comment:
Suggestion: to keep these names and variables succinct, consider simply saying `toggles`, rather than `feature toggles`.  That is how we chose to name our new github repo: https://github.com/edx/edx-toggles

Reply:
I'm not a fan of the long name, but i used it originally because I was trying to follow [OEP-0017](https://open-edx-proposals.readthedocs.io/en/latest/oep-0017-bp-feature-toggles.html#req-12-report). In the OEP there's a bunch of different types of toggles. Notably, the doc goes to some length to differentiate "experiment toggles" and puts them out of scope. 

I wasn't involved with the writing of the OEP, so I didn't want to stray too far from it's naming scheme. But if I took that too far, I'd like to see the tokens get shorter.